resource_customizations/argoproj.io/AnalysisRun/health.lua

```diff
@@ -14,15 +14,15 @@ if obj.status ~= nil then
     end
     if obj.status.phase == "Failed" then
         hs.status = "Degraded"
-        hs.message = "Analysis run failed"
```

Comment:
We need to support the previous rollouts controller version that does not store the message in Rollout status. Please return a hardcoded message if status does not have real message .